### ISSUE TITLE
Update KPI numbering

### ISSUE THREAD

**Issue by AmaniKrayemRCPCH** [rcpch/rcpch-audit-engine#409]
Apologies, I delayed in responding to the last issue so I didn't provide the correct numbering in time. 

Correct numbering is below. 3a, 9a, 9b, and all of the submetrics under 9a and 9b need updating.  
The title for 9b.f needs updating too. 

Perhaps it might be more clear if we indicate which ones are sub-metrics (the items below 9a. and 9b.) Maybe by removing the bold formatting in the first column for those sub-metrics?

````
1. Paediatrician with expertise in epilepsies
2. Epilepsy Specialist Nurse
3a. Tertiary input
3b. Epilepsy surgery referral
4. ECG
5. MRI
6. Assessment of mental health issues
7. Mental health support
8. Sodium Valproate
9a. Comprehensive care planning agreement
  a. Patient-held individualised epilepsy document/copy of clinic letter that includes care planning information
  b. Patient/carer/parent agreement to the care planning
  c. Care planning has been updated when necessary. 
9b. Comprehensive care planning content
  a. Parental prolonged seizures care plan
  b. Water safety
  c. First aid
  d. General participation and risk
  e. Service contact details
  f. Sudden unexpected death in epilepsy
10. School individualised healthcare plan. 
````

**Comment by anchit-chandran** [comment]
I believe this has been fixed



![Image](https://user-images.githubusercontent.com/68898006/232775146-01b89ac1-08b7-437c-b98c-90332b0fb567.png)

**Comment by pacharanero** [comment]
It looks like 9e and 9f are switched around compared to the last information we had on this @AmaniKrayemRCPCH 
compare with https://github.com/rcpch/rcpch-audit-engine/issues/375

**Comment by AmaniKrayemRCPCH** [comment]
Apologies - e is SUDEP and f is Service Contact Details,

Can we update the submetrics so theyre listed as **a. b. c.** rather than **9a. 9b. 9c.** 

If you feel that's not clear enough, we can include the PI number - **9a.a, 9a.b, 9a.c**

**Comment by anchit-chandran** [comment]
Update this to roman numeral numbering, as per discussion today with Amani:

```
9A
9i
9ii
...
9B
9i
9ii
...
```

Leave 3. and 3b. as they are.

Update tinting of sub-metrics according to RCPCH preference @nikyraja @AmaniKrayemRCPCH to provide

**Comment by eatyourpeas** [comment]
closed by #452
